### DISCUSSION TITLE
how to customise the colours of the pie chart ? please

### DISCUSSION THREAD

**Discussion by assia-zoubir** [Budibase/budibase#13534] [Help]
hallo, please can someone help me. here my pieChart, i would like to have the same color when it's free Time, is it possible with css, javascript. it's not correct to have a different color for the same things. how can i do that please ?
<img width="435" alt="image" src="https://github.com/Budibase/budibase/assets/149948208/15f22c93-5fe4-4ed5-9df5-3e46662dd19f">

**Comment by mikesealey** [comment]
Hi @assia-zoubir I've been looking into this for you. 

It's currently not possible to set the colours of specific segments, only to select 5 colours that will repeat around your pie chart. 
![image](https://github.com/Budibase/budibase/assets/39269380/2779d4c4-cb68-49f0-9518-80c5e8ca6bb8)

I've raised a [feature request ](https://github.com/Budibase/budibase/issues/13554) on your behalf that will allow some additional flexibility on this.

The chart you've shared as an example seems to be set up more like a timeline, is a pie chart the right tool for displaying your data? Would it be more appropriate to combine your Free Time entries into one?
